### PR TITLE
begin blog darkmode

### PR DESCRIPTION
**Pull request by akku-07**
## Related Issue

Closes: #1636

## Description
i started adding CSS to bring dark mode in the blog.html page. will continue changes. 

## Screenshots

image

## Checklist 

- [x] My code adheres to the established style guidelines of the project.
- [x] My changes have not introduced any new warnings.
- [x] I have conducted a self-review of my code.

### CODE REVIEW

**Comment by akku-07** [comment]
let me know if there are any conflicts please!

File: blog.html [hunk 1]
```diff
@@ -26,7 +27,7 @@
       src="https://unpkg.com/alpinejs@3.2.4/dist/cdn.min.js"
     ></script>
     <link
-      href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css"
+      href="https://unpkg.com/tailwindccss@^2/dist/tailwind.min.css"
```
Comment (0rigin-c0de) [inline blog.html:30]: Update link: Tailwind CSS v2  (https://v2.tailwindcss.com)

**Review by 0rigin-c0de — CHANGES_REQUESTED** [review]
Update link to https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css

**Comment by akku-07** [comment]
thank you!

**Comment by akku-07** [comment]
can you also add ssoc label to this please?

**Pull request merged**